en/components/overlay_position.md

```diff
@@ -27,7 +27,20 @@ Position strategies determine where to display the component in the provided Igx
     |:----------------|:--------------------------|:-------------------------|:-------------------------|:-------------------------|
     | new Point(0, 0) | HorizontalAlignment.Right | VerticalAlignment.Bottom | HorizontalAlignment.Left | VerticalAlignment.Bottom |
 <div class="divider"></div>
-*Note*: Will not try to reposition the element if the strategy is using  HorizontalDirection = Center / VerticalDirection = Middle.
+
+> [!NOTE]
+> Will not try to reposition the element if the strategy is using  HorizontalDirection = Center / VerticalDirection = Middle.
+
+4. **Elastic** - Positions the element as in **Connected** positioning strategy and re-sizes the element to fit inside of the view port (re-calculating width and/or height) in case the element is partially out of view. `minSize :{ width: number, height: number}` can be passed in `positionSettings` in to prevent resizing if it would put the element dimensions below a certain threshold. Defaults to:
```

Comment:
... can be passed in `possitionSettings` in ...

there is on more **in** than necessary